rotkehlchen/tests/api/test_data_purging.py

```diff
@@ -212,7 +213,11 @@ def _assert_zksynclite_txs_num(cursor: 'DBCursor', tx_num: int, swap_num: int) -
 
     # check that removing bitcoin events works
     events_db = DBHistoryEvents(rotki.data.db)
-    event_identifier1, event_identifier2 = 'xxxx', 'zzzz'
+    event_identifier1, event_identifier2, event_identifier3 = (
+        f'{BTC_EVENT_IDENTIFIER_PREFIX}xxxx',
```

Comment:
can you please use real BTC hashes and actually turn them into variables and use them below. Otherwise it's hard to understand the test